### PR TITLE
push / pull media improvements

### PR DESCRIPTION
**Pull request by Chive**

### CODE REVIEW

**Comment by intellectronica** [comment]
:+1:

File: aldryn_client/localdev/main.py [hunk 1]
```diff
@@ -197,7 +196,8 @@ def download_media(client, path=None):
         with tarfile.open(fileobj=media_fobj, mode='r:gz') as media_archive:
             media_archive.extractall(path=path)
 
-        click.secho('Downloaded media files into {}'.format(path), fg='green')
+    os.remove(backup_path)
```
Comment (jqb) [inline aldryn_client/localdev/main.py]: funny ... I was always using `os.unlink`. Apparently those functions are doing the same.

File: aldryn_client/localdev/main.py [hunk 2]
```diff
@@ -239,7 +239,9 @@ def upload_media(client):
 
     click.secho('Creating archive of local media folder', fg='green')
     with tarfile.open(archive_path, mode='w:gz') as tar:
-        tar.add(os.path.join(project_home, 'data/media'), arcname='media')
+        media_dir = os.path.join(project_home, 'data/media')
```
Comment (jqb) [inline aldryn_client/localdev/main.py]: (Just a hint). Maybe it's not *that* important. But if you're already using `os.path.join` you could make it really platform dependent: `os.path.join(project_home, 'data', 'media')`

**Comment by jqb** [comment]
LGTM

File: aldryn_client/localdev/main.py (hunk 2 above)
Comment (Chive) [inline aldryn_client/localdev/main.py]: ``True``, thanks!

**Pull request merged**